static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -0,0 +1,3 @@
+.DevHub-MyAddons-copy {
+  max-width: 32em;
```

Comment:
Not according to UX, no. I do want it to be responsive as a stretch goal though.